### PR TITLE
Use pytest log_cli_level for openshift-installer log level

### PR DESCRIPTION
**Pull request by clacroix12**
Signed-off-by: Coady LaCroix <clacroix@redhat.com>

### CODE REVIEW

**Comment by clacroix12** [comment]
I have tested deployment as well as cluster destroy and both processes passed without issue. I can post logs if necessary.

File: tests/conftest.py [hunk 1]
```diff
@@ -383,3 +384,15 @@ def environment_checker(request):
...
+@pytest.fixture(scope="session")
+def log_cli_level(pytestconfig):
+    """
+    Retrieves the log_cli_level set in pytest.ini
+
+    Returns:
+        str: log_cli_level set in pytest.ini or DEBUG if not set
+
+    """
+    return pytestconfig.getini('log_cli_level') or 'DEBUG'
```
Comment (vasukulkarni) [inline tests/conftest.py]: can you test if it respects the cli option that pytest provides?

**Comment by vasukulkarni** [comment]
Overall LGTM

File: tests/conftest.py (hunk 1 above)
Comment (clacroix12) [reply tests/conftest.py]: @vasukulkarni yes I did, and it does

**Pull request merged**